scripts/npm-audit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+HIGH_VULN_DEV="$(grep -o -i High vulnerabilities.txt | wc -l)"
```

Comment:
Nit-pick, if you both fix one high vulnerability and introduce a new one, it would not get caught, would it?

Reply:
I think you need to grep for Critical as well. Or maybe only critical?

Reply:
Yeah it wouldn't get caught